components/form/BannerSettings.vue

```diff
@@ -0,0 +1,133 @@
+<script>
+import Checkbox from '@/components/form/Checkbox';
```

Comment:
The banner is only updated on the first apply:

- Go to the brand setting page
- Change the alignment of one of the banners
- Click Apply to save the change
- The banner updates to show the change
- Change the alignment again
- Click Apply to save the change
- The banner does not update to show the new alignment

Reply:
@nwmac I had to fix which v-model was being targeted in BannerSettings from the `data` object to `value[bannerType]`. Now it updates the value as intended.